torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -140,6 +140,22 @@ template std::shared_ptr<OwnerRRef<IValue>> RRefContext::getOrCreateOwnerRRef<
 template std::shared_ptr<OwnerRRef<py::object>> RRefContext::
     getOrCreateOwnerRRef<py::object>(const RRefId& rrefId);
 
+template <typename T>
+std::shared_ptr<OwnerRRef<T>> RRefContext::createOwnerRRef() {
```

Comment:
I assume that this does not take a `T` argument as we can also use this to support run `rpc.remote` on self?

Reply:
I agree that we can use this to run rpc.remote on self, but I don't follow how it would be possible to remove the T argument -- I need to pass it to the constructor of OnwerRRef.

Reply:
Oh, I mean I was thinking this API should take a `T&& value` for now as this is how we use it currently, i.e., something like:

```c++
template <typename T>
std::shared_ptr<OwnerRRef<T>> RRefContext::createOwnerRRef(T&& value) {
```

Reply:
Yes i'm thinking of a use case where we don't set the value right away.